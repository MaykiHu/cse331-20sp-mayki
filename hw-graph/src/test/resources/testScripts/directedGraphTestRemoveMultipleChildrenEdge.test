#Create a graph
CreateGraph graph1

#Add three non-same nodes to the graph
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

#Add a pair of edges with the same parent to the graph
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n3 e2

#Removes a pair of edges with the same parent from the graph
RemoveEdge graph1 n1 n2 e1
RemoveEdge graph1 n1 n3 e2