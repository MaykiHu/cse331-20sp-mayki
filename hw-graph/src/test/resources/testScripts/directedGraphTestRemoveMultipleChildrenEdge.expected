#Create a graph
created graph graph1

#Add three non-same nodes to the graph
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

#Add a pair of edges with the same parent to the graph
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n3 in graph1

#Removes a pair of edges with the same parent from the graph
removed edge e1 from n1 to n2 in graph1
removed edge e2 from n1 to n3 in graph1